src/pages/sighting/encounters/Encounters.jsx

```diff
@@ -210,6 +213,39 @@ export default function Encounters({
             value: schema.getValue(schema, encounter),
           }),
         );
+
+        const individualGuid = encounter?.individual?.guid;
+        const individualName = deriveIndividualName(
+          encounter?.individual,
+        );
```

Comment:
Should this have a fallback? If an identified individual's name can't be derived, there is no text that links to that individual's page.

Reply:
Oh wait. I don't think you can create and individual without giving it a name. I think the form validation prevents you from submitting. So maybe a moot point.

Reply:
I agree that it should not be possible to create an individual from the front end without giving it a name. I think that some migrated zebras do not have first names though.